src/contracts/core/DelegationManager.sol

```diff
@@ -91,17 +91,14 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
      * @param metadataURI is a URI for the operator's metadata, i.e. a link providing more details on the operator.
      *
      * @dev Once an operator is registered, they cannot 'deregister' as an operator, and they will forever be considered "delegated to themself".
-     * @dev This function will revert if the caller attempts to set their `earningsReceiver` to address(0).
+     * @dev This function will revert if the caller is already delegated to an operator.
      * @dev Note that the `metadataURI` is *never stored * and is only emitted in the `OperatorMetadataURIUpdated` event
      */
     function registerAsOperator(
         OperatorDetails calldata registeringOperatorDetails,
         string calldata metadataURI
     ) external {
-        require(
-            _operatorDetails[msg.sender].earningsReceiver == address(0),
-            "DelegationManager.registerAsOperator: operator has already registered"
-        );
```

Comment:
I think we should default to setting the value of this deprecated field to zero.

We dont need to keep this require here, but if we plan to repurpose this field later, we need to make sure people aren't already setting it to a nonzero value.

Reply:
it has already been set to a nonzero value by all existing users 😞 
one option I suggested was setting it to the operator's own address, but I think setting it to zero could also be fine.

Reply:
Since all existing registered operators have nonzero values I thought its best to not make assumptions on whatever value this field is and let it be whatever people have it set to.

If we use this field later, we need to allow for a period for operators to update it and then assume afterwards that it's configured.

Reply:
ah right i missed that we currently require this to be nonzero. yikes!

if we use this field later... we could do that or we might want to migrate and set it to 0. or set it to a sane default.